AudioKit/Common/Nodes/Effects/Dynamics/Compressor/AKCompressor.swift

```diff
@@ -114,22 +115,17 @@ open class AKCompressor: AKNode, AKToggleable, AUEffect, AKInput {
         self.releaseTime = releaseTime
         self.masterGain = masterGain
 
-        input?.connect(to: inputGain)
         inputGain.volume = 0
-        mixer = AKMixer(inputGain)
-
-        effectGain = AKMixer(input)
```

Comment:
This was the issue, input connects to both inputGain and effectGain on init.  inputGain was the inputNode, so connection after instantiation wouldn't route through the effect.